src/instructlab/model/train.py

```diff
@@ -1095,42 +1114,42 @@ def _run_phased_training(
 
         phase_model.ended_at_utc = TrainingJournal.now_utc()
 
-        journal.current_phase = TrainingPhases.EVAL1
+        journal.current_phase = TrainingPhases.TRAIN2
         journal.commit()
 
         logger.debug("Finished training #1\n%s", journal.print_model_rich())
     else:
         click.secho("SKIPPING: Training Phase 1/2; already in Journal", fg="cyan")
 
-    if journal.current_phase == TrainingPhases.EVAL1:
-        click.secho("MMLU evaluation for Phase 1...", fg="cyan")
+    # if journal.current_phase == TrainingPhases.EVAL1:
+    #     click.secho("MMLU evaluation for Phase 1...", fg="cyan")
 
-        # NOTE: requires hf_format sub-directory. Training sets this up.
-        phase1_checkpoints_dir = phase1_checkpoints_dir / "hf_format"
+    # NOTE: requires hf_format sub-directory. Training sets this up.
+    # phase1_checkpoints_dir = phase1_checkpoints_dir / "hf_format"
 
-        phase_model = journal.journal.eval_1
-        if phase_model is None:
-            # if it's not None, it already exists and may have 'results', so we shouldn't overwrite it.
-            phase_model = EvalPhaseModel(
-                checkpoints=list(phase1_checkpoints_dir.iterdir())
-            )
-            journal.journal.eval_1 = phase_model
-        journal.commit()
+    # phase_model = journal.journal.eval_1
+    # if phase_model is None:
+    #     # if it's not None, it already exists and may have 'results', so we shouldn't overwrite it.
+    #     phase_model = EvalPhaseModel(
+    #         checkpoints=list(phase1_checkpoints_dir.iterdir())
+    #     )
+    #     journal.journal.eval_1 = phase_model
+    # journal.commit()
 
-        best_checkpoint = _evaluate_dir_of_checkpoints(
-            eval_func=_mmlu, phase_model=phase_model, journal=journal
-        )
+    # best_checkpoint = _evaluate_dir_of_checkpoints(
+    #     eval_func=_mmlu, phase_model=phase_model, journal=journal
+    # )
 
-        phase_model.best_checkpoint = best_checkpoint
-        phase_model.ended_at_utc = TrainingJournal.now_utc()
+    # phase_model.best_checkpoint = best_checkpoint
+    # phase_model.ended_at_utc = TrainingJournal.now_utc()
 
-        journal.current_phase = TrainingPhases.TRAIN2
-        journal.commit()
-        logger.debug("Finished eval #1\n%s", journal.print_model_rich())
-    else:
-        click.secho(
-            "SKIPPING: MMLU evaluation for Phase 1; already in Journal", fg="cyan"
-        )
+    #     journal.current_phase = TrainingPhases.TRAIN2
+    #     journal.commit()
+    #     logger.debug("Finished eval #1\n%s", journal.print_model_rich())
+    # else:
+    #     click.secho(
+    #         "SKIPPING: MMLU evaluation for Phase 1; already in Journal", fg="cyan"
+    #     )
```

Comment:
is commenting the code out necessary? seems to me that setting `journal.current_phase` to `TRAIN2` effectively skips the mmlu check anyway. I think that coupled with a comment explaining why we are skipping to train 2 + maybe a debug log statement about it should be enough

Reply:
Mostly because it's a hotfix- I don't want to scrape the logic from the codebase in case other people are interested in it in the immediate future.

Reply:
im arguing for even fewer changes to it than you lol
im not asking for it to be removed - im suggesting to leave it uncommented and steer control away from it via the current_phase - that's it
just cause im not a fan of the idea of shipping commented out code in an actual release 

not blocking tho

Reply:
OH gotcha, yeah I just want it commented so that it's clear that it's not the codepath being used.